packages/rum-core/test/performance-monitoring/performance-monitoring.spec.js

```diff
@@ -796,4 +797,35 @@ describe('PerformanceMonitoring', function() {
 
     tr.end()
   })
+
+  if (window.EventTarget) {
```

Comment:
Need a test for this scenario click triggers a history `pushstate` change and have to check transaction details expected.